### PR TITLE
Update application.js

### PR DESCRIPTION
**Pull request by ochagata**
Solves #230

### CODE REVIEW

File: lib/rubycritic/generators/html/assets/javascripts/application.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ $("#churn-vs-complexity-graph-container").highcharts({
   },
   plotOptions: {
     series: {
-      turboThreshold: 1500
+      turboThreshold: 15000
```
Comment (Onumis) [inline lib/rubycritic/generators/html/assets/javascripts/application.js]: I'd rather set this to `0` so this won't ever be a problem.

I'm also thinking on how we could make this configurable :thinking: 
Maybe add a control on the page to set this value and re-render the chart. But not sure if it'd be that useful. 

@AzizHarrington what do you think?

**Comment by ochagata** [comment]
@Onumis Made the change

**Comment by ochagata** [comment]
@Onumis mind merging this, please?

**Comment by Onumis** [comment]
will make a release later

**Comment by ochagata** [comment]
@Onumis Awesome

**Pull request merged**